News documentation US details & edit (#116)
exoplatform/exo-documentation: docs/GettingStarted.rst

|image161|

You can access the content of the article either by clicking on its title or by clicking on "read more".

> The details of the article are available on the current page and display all the information that has been entered as well as publication date and user.

Reviewer: ```suggestion
The details of the article are available on the current page and display all the information, including the publication date and the author.
```

|image162|

You can change the content of the article using the edit icon. 

> The possible actions from the edit mode interface are "update", update and post" and "cancel". The buttons update and update and post are disabled until changes are made.

Reviewer: ```suggestion
The possible actions from the edit mode interface are `Update`, `Update and post` and `Cancel`. The buttons `Update` and `Update and post` are disabled until changes are made.
```

> The update action allows you to make changes in the article without reposting it unlike the "update and post" action that allows to apply the changes in the article details and permits to rise up the article's preview in the activity stream.

Reviewer: ```suggestion
The `Update` action allows you to make changes in the article without reposting it unlike the `Update and post` action that allows to apply the changes in the article details and rise up the article's preview in the activity stream.
```

|image163|

> After updating the article, the information update user, update date and hour are available in the details as well as the different changes.

Reviewer: ```suggestion
When an article has been edited, the details view display the updated date and author besides initial information.
```
